src/datepicker/datepicker.stories.ts

```diff
@@ -7,13 +7,86 @@ import {
 	text,
 	boolean
 } from "@storybook/addon-knobs/angular";
+import {
+	FormBuilder,
+	FormGroup,
+	Validators,
+	FormsModule,
+	ReactiveFormsModule
+} from "@angular/forms";
+import {
+	Component,
+	Input,
+	Output,
+	EventEmitter
+} from "@angular/core";
 import { DatePickerModule } from "../";
 
+
+@Component({
+	selector: "app-date-picker",
+	template: `
+		<form [formGroup]="rForm">
+			<ibm-date-picker
+				formControlName="single"
+				label="Date Picker Label"
+				invalidText="Invalid date format"
+				[invalid]="invalidSingle"
+				(valueChange)="valueChange.emit($event)">
+			</ibm-date-picker><br>
```

Comment:
Beneath each of these, in a `<code>` block, could we write out the `.value`? something like `rForm.controls["single"].value | json` should work